tests/func-tests/monitoring_test.go

```diff
@@ -38,6 +41,9 @@ import (
 var runbookClient = http.DefaultClient
 
 const (
+	noneImpact float64 = iota
+	warningImpact
+	criticalImpact
```

Comment:
Please don't mix `iota` consts with other consts. Can you split it into two const sentenses?

Reply:
Done